lib/matplotlib/_enums.py

```diff
@@ -79,6 +69,12 @@ class JoinStyle(str, _AutoStringNameEnum):
 
     """
 
+    def _generate_next_value_(name, start, count, last_values):
+        return name
+
+    def __hash__(self):
+        return str(self).__hash__()
+
```

Comment:
Do we really _need_ these in all of the classes?

* Can we just explicitly list the string enumeration rather than auto? `miter = "miter"` (a bit odd we didn't go with caps for the enum `JointStyle.MITER` is typically how I see them written...
* Does `__hash__` not get string's hash automatically from subclassing?

Reply:
"all" is only two, but this also seems to have also not worked (other attempts failed on import...this imported but made the tests fail).